clients/admin-ui/src/features/datamap/SettingsBar.tsx

```diff
@@ -28,11 +29,19 @@ const SettingsBar = () => {
   const { tableInstance } = useContext(DatamapTableContext);
   const { systemsCount: totalSystemsCount, dictionaryService: compassEnabled } =
     useFeatures();
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const rows = tableInstance?.getRowModel().rows || [];
+  const uniqueSystemKeysFromFilteredRows = useMemo(
+    () => uniq(rows?.map((row) => row.original["system.fides_key"])),
+    [rows],
+  );
```

Comment:
You can avoid the need for disabling the eslint warning by making a few adjustments
```suggestion
  const rowModel = tableInstance?.getRowModel();
  const uniqueSystemKeysFromFilteredRows = useMemo(() => {
    const rows = rowModel?.rows || [];
    uniq(rows?.map((row) => row.original["system.fides_key"]));
  }, [rowModel]);
```